src/lib/eth-utils.js

```diff
@@ -106,3 +107,14 @@ export function toFraction(amount, decimals, separator = '.') {
 
   return whole + (fraction ? separator + fraction : '')
 }
+
+/**
+ * @param {bigint | number} gasLimit
+ * @param {number} multiplier
+ * @returns {bigint} Increased fee
+ */
+export function increaseFee(gasLimit, multiplier = INCREASE_FEE_MULTIPLIER) {
+  const increasedGasLimit = BigNumber(Number(gasLimit)).multipliedBy(multiplier).toNumber()
```

Comment:
Please explain why. The argument can be either a `number` or `bigint` type. I used Number() to convert `bigint -> number`.

Reply:
Does the `BigNumber()` accept `Bigint` as an argument?

Reply:
Why do we use `bignumber.js` if `Bigint` is common?

Reply:
> Does the `BigNumber()` accept `Bigint` as an argument?

Officially not. Nothing about that in the documentation. The TS [types](https://github.com/MikeMcl/bignumber.js/blob/master/bignumber.d.ts#L327) says that it does not accept `bigint`. Even if it worked I wouldn't use it, because it can break the app on the next `bignumber.js` lib update. There is a [task](https://trello.com/c/rZeUAbo2/445-replace-bignumberjs-with-an-alternative-bigjs) regarding this.

Reply:
> Why do we use `bignumber.js` if `Bigint` is common?

These are different things. They cannot replace each other.

`BigNumber` is a library that does arithmetic operations on floating point numbers.
`BigInt`  is builtin feature in JS for integer numbers

Reply:
@martiliones 
Comment or resolve